docs/content/integrations/airlift/tutorial.mdx

```diff
@@ -1 +1,1258 @@
-Empty for now, referenced in reference.mdx.
\ No newline at end of file
+## Example: Migrating an Airflow DAG to Dagster
+
+This example demonstrates how to migrate an Airflow DAG to Dagster using the `dagster-airlift` package. It contains code examples of how to peer, observe, and migrate assets from an Airflow DAG to Dagster. The below guide will walk through each of these steps in detail.
```

Comment:
Apologies if I missed the context on this.

The plan is to keep it in its own package, and not move it to `dagster-airflow` under something like `dagster_airflow.airlift`?

Reply:
For now, we'll be keeping it as its own package. Once we've validated airlift direction, we'll consolidate.